webapp/fmprod/src/main/java/org/apache/oodt/cas/product/jaxrs/services/CasProductJaxrsService.java

```diff
@@ -40,10 +41,7 @@
 import java.util.logging.Logger;
 
 import javax.servlet.ServletContext;
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
-import javax.ws.rs.QueryParam;
+import javax.ws.rs.*;
```

Comment:
We shouldn't be using * imports. Importing a code formatter as mentioned in an earlier comment and reformatting these files will fix that. Or you will have to manually change settings in IDEA.